operators/hpe-csi-operator/2.1.1-0/manifests/hpe-csi-operator.v2.1.1-0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,349 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: hpe-csi-operator.v2.1.1-0
+  namespace: placeholder
+  annotations:
+    operatorhub.io/ui-metadata-max-k8s-version: "1.23"
+    alm-examples: >-
+      [
+        {
+          "apiVersion": "storage.hpe.com/v1",
+          "kind": "HPECSIDriver",
+          "metadata": {
+            "name": "csi-driver"
+          },
+          "spec": {
+            "disable": {
+               "nimble": false,
+               "primera": false,
+               "alletra6000": false,
+               "alletra9000": false
+            },
+            "imagePullPolicy": "IfNotPresent",
+            "logLevel": "info",
+            "disableNodeConformance": false,
+            "iscsi": {
+              "chapUser": "",
+              "chapPassword": ""
+            },
+            "registry": "quay.io",
+            "kubeletRootDir": "/var/lib/kubelet/",
+            "disableNodeGetVolumeStats": false
+          }
+        }
+      ]
+    categories: Storage
+    certified: 'true'
+    createdAt: 'Wed Jun 16 11:30:38 PDT 2021'
+    description: >-
+      A Container Storage Interface (CSI) driver for HPE storage platforms. The CSI driver
+      allows you to use HPE storage with your preferred container orchestrator.
+    containerImage: 'quay.io/hpestorage/csi-driver-operator:v2.1.1'
+    support: 'HPE'
+    capabilities: Basic Install
+    repository: 'https://github.com/hpe-storage/co-deployments'
+spec:
+  displayName: HPE CSI Operator for Kubernetes
+  description: >
+    The HPE CSI Operator for Kubernetes packages, deploys, manages, upgrades the HPE CSI Driver on
+    Kubernetes and OpenShift for dynamic provisioning of persistent volumes on HPE storage systems.
+
+    The HPE CSI Driver for Kubernetes leverages HPE storage
+    platforms to provide scalable and persistent storage for stateful
+    applications. Please refer to our CSI driver [documentation](https://scod.hpedev.io/csi_driver/index.html)
+    for supported HPE platforms and full list of supported CSI features.
+
+    ## Installation
+
+    Refer to the HPE CSI Operator for Kubernetes [documentation](https://scod.hpedev.io/csi_driver/deployment.html#operator).
+
+
+  maturity: stable
+  version: 2.1.1-0
+  replaces: hpe-csi-operator.v2.1.0
+  minKubeVersion: 1.20.0
+    keywords:
```

Comment:
`keywords` should be intended one level to the left

Reply:
Thanks